workers/social/lib/social/error.coffee

```diff
@@ -4,9 +4,11 @@ KnownErrors =
 
 module.exports = class KodingError extends Error
 
+  encoder   = require 'htmlencode'
```

Comment:
unless you have a valid reason, please take `require` line on top of the file and fix spacing between the variable name and the equal sign.